terraform/accounts/main/main.tf

```diff
@@ -22,6 +26,7 @@ module "iam_common" {
 module "iam_users" {
   source                          = "../../modules/iam-users"
   admins                          = "${var.admins}"
+  backups                         = "${var.admins}"                                                // Add admins to backups group, gives 2nd line access to backups
```

Comment:
Wasn't this going to be the same as `prod_infrastructure_users`?

Reply:
Good call, admins can assume anything anyway (tested) 👍

Reply:
Nice. Good to know.